app/components/Texting/Texting.jsx

```diff
@@ -0,0 +1,223 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import './Texting.scss';
```

Comment:
Do you think you could import this as a [CSS module](https://github.com/css-modules/css-modules) rather than a global CSs import? We use a library called CSS Modules, which allows us to write CSS files that are all locally scoped rather than globally scoped. This helps us avoid naming conflicts between different CSS files, since you can freely reuse names across CSS modules.

When imported into JavaScript, you can access each CSS class as a property on the imported object, and you are expected to pass them into the `className` property on each HTML element rather than just passing in a standalone string. See https://github.com/ShelterTechSF/askdarcel-web/blob/001e9a039787b9cdfa2d7142672ed0a925320d26/app/components/ui/Navigation/Navigation.jsx#L6 for an example of use using CSS modules. Also, you will need to update webpack.config.js to add this file to the list of files that should be treated as CSS modules: https://github.com/ShelterTechSF/askdarcel-web/blob/001e9a039787b9cdfa2d7142672ed0a925320d26/webpack.config.js#L134-L147